src/components/NotesContainer/NoteList/Note/Note.tsx

```diff
@@ -1,20 +1,24 @@
 import DeleteIcon from '@mui/icons-material/Delete';
 import { IconButton } from '@mui/material';
 import React from 'react';
-import { useDispatch } from 'react-redux';
 import { Link } from 'react-router-dom';
-import { deleteNote } from 'redux/notes-slice';
 
+import { useAppDispatch } from 'components/hooks/redux';
 import styles from 'components/NotesContainer/NoteList/Note/Note.module.scss';
 import { NoteProps } from 'components/NotesContainer/NoteList/Note/Note.types';
+import { fetchNotes, removeNote } from 'store/actions';
 
 const Note: React.FC<NoteProps> = ({ note }) => {
-  const dispatch = useDispatch();
+  const dispatch = useAppDispatch();
+  const onSubmit = (values: string) => {
```

Comment:
it is not submit
`onRemoveButtonClick` or `removeNoteHandler` smth like that

Reply:
you are right, ready